snooty/rstspec.toml

```diff
@@ -641,6 +641,53 @@ example = """.. card-group::
    ${0: Card content}
 """
 
+[directive."landing:product"]
+content_type = "block"
```

Comment:
similar here too. don't we want an argument to parse for heading? this is accepting a heading arg [in the doc repo](https://github.com/schmalliso/docs-landing/blob/3099d7975ecfa8f1f018034b35ba7b8a00af5416/source/index.txt#L51C25-L51C48)  but getting parsed as a paragraph and being [styled as a header](https://github.com/mongodb/snooty/pull/885/files#diff-9172bb560b5a007439e019059cc60578677f90ee536366b794f256e9bee055d5R30)